src/entity/sources/SourceReport.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, Index, ManyToOne, PrimaryColumn } from 'typeorm';
+import type { User } from '../user';
+import { ReportReason } from '../common';
+
+@Entity()
```

Comment:
Do we want to go seperate route, or stick with childEntities?
(Probably was a discussion I can't recall)

Reply:
Was near the end of the RFC, but yeah, it was decided to go similar to vote entities.